nano/scripts/libraries/0_jquery.js

```diff
@@ -8513,7 +8513,7 @@ var rvalidbraces = /(?:^|:|,)(?:\s*\[)+/g;
 var rvalidescape = /\\(?:["\\\/bfnrt]|u[\da-fA-F]{4})/g;
 var rvalidtokens = /"[^"\\\r\n]*"|true|false|null|-?(?:\d+\.|)\d+(?:[eE][+-]?\d+|)/g;
 
-jQuery.parseJSON: function( data ) {
+jQuery.parseJSON = function( data ) {
```

Comment:
I have absolutely no fucking clue how this was working, because `:` is not in any way valid here